amulet/helpers.py

```diff
@@ -65,9 +68,9 @@ def juju(args, env=None, include_model=True):
             if arg.startswith(model_flag):
                 break
         else:
-            args = list(args)
-            # insert the model arg after the command, but before any other args
-            args[1:0] = [model_flag, default_environment()]
+            if env is None:
+                env = os.environ
+            env['JUJU_ENV'] = env['JUJU_MODEL'] = default_environment()
```

Comment:
The `model_flag` is still used to check if an explicit model / env has been passed in via the args.  Though, the CLI param would override the env, anyway, so maybe we can just collapse it.